### PR TITLE
[FOR-2023] - Add event Fortaleza

### PR DESCRIPTION
**Pull request by renat473**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

### CODE REVIEW

File: data/events/2023-fortaleza.yml [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2023-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate:  2023-09-23T08:00:00-03:00  # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate:  2023-09-23T17:00:00-03:00  # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: 2021-10-14T10:00:00-03:00 # start accepting talk proposals.
+cfp_date_end: 2021-11-15T07:00:00-03:00 # close your call for proposals.
+cfp_date_announce: 2021-10-14T08:00:00-03:00 # inform proposers of status
```
Comment (phrawzty) [inline data/events/2023-fortaleza.yml:19]: These dates are in 202**1** (and probably the wrong months?)

File: data/events/2023-fortaleza.yml [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+cfp_open: "false"
```
Comment (phrawzty) [inline data/events/2023-fortaleza.yml:21]: _If_ your CFP is open, this should be set to `true`.

File: data/events/2023-fortaleza.yml [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+
+# Location
+#
+coordinates: "3.7697702, -38.4799603" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "Local: UNIFOR - Universidade de Fortaleza - Fortaleza/CE" # Defaults to city, but you can make it the venue name.
+#
+location_address: "Av. Washington Soares, 1321 - Edson Queiroz" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  - name: propose
```
Comment (phrawzty) [inline data/events/2023-fortaleza.yml:38]: _If_ your CFP isn't open, this should be commented out.

File: content/events/2023-fortaleza/welcome.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        {{< event_link page="sponsor" text="Apoiei o Devopsdays Fortaleza 2023!" >}}
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-2">
+        <strong>CFP</strong>
+      </div>
+      <div class="col-md-6">
+        {{< event_link url-key="cfp_link" text="Submeta a sua talk!" >}}
+      </div>
```
Comment (phrawzty) [inline content/events/2023-fortaleza/welcome.md:57]: _If_ your CFP isn't open, this should be commented out.

File: data/events/2023-fortaleza.yml [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+
+registration_date_start: 2023-06-29 # start accepting registration. Leave blank if registration is not open yet. This will make the "Register" button appear on your "Welcome" page.
+registration_date_end: 2023-08-10 # close registration. Leave blank if registration is not open yet. If you set "registration_date_start" you need a value here.
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "https://devopsdaysce.eventize.com.br" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+coordinates: "3.7697702, -38.4799603" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
```
Comment (phrawzty) [inline data/events/2023-fortaleza.yml:32]: These coordinates appear to be in the ocean?

image

File: data/events/2023-fortaleza.yml [hunk 6]
```diff
@@ -0,0 +1,259 @@
...
+#
+location_address: "Av. Washington Soares, 1321 - Edson Queiroz" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  - name: propose
+  - name: location
+  - name: registration
+    url: https://devopsdaysce.eventize.com.br
+ # - name: program
+  - name: speakers
```
Comment (phrawzty) [inline data/events/2023-fortaleza.yml:43]: Comment this out until you have speakers.
```suggestion
  # - name: speakers
```

**Review by phrawzty — CHANGES_REQUESTED** [review]

File: data/events/2023-fortaleza.yml (hunk 1 above)
Comment (renat473) [reply data/events/2023-fortaleza.yml]: I correct the date

File: data/events/2023-fortaleza.yml (hunk 2 above)
Comment (renat473) [reply data/events/2023-fortaleza.yml:21]: set to false

File: content/events/2023-fortaleza/welcome.md (hunk 4 above)
Comment (renat473) [reply content/events/2023-fortaleza/welcome.md:57]: is inactive and commented the line of the indicated file.

File: data/events/2023-fortaleza.yml (hunk 3 above)
Comment (renat473) [reply data/events/2023-fortaleza.yml]: is inactive and commented the line of the indicated file.

File: data/events/2023-fortaleza.yml (hunk 5 above)
Comment (renat473) [reply data/events/2023-fortaleza.yml]: fix update coordinates

**Pull request merged**